test/setup.js

```diff
@@ -0,0 +1,20 @@
+import sinon from 'sinon';
+
+const levels = ['log', 'debug', 'info', 'warn', 'error'];
+
+before(() => {
+  for (let l of levels)
+    if (!console[l].restore)
+      sinon.stub(console, l);
+})
+
+beforeEach(() => {
+  for (let l of levels)
+    console[l].reset();
+})
+
+
+after(() => {
+  for (let l of levels)
+    console[l].restore();
+})
```

Comment:
could you  please add a doc string (maybe at the top of file), regarding suppressing console logs in setup?

Reply:
yes ,good idea.